dev_guide/deployments.adoc

```diff
@@ -191,6 +191,53 @@ To see what the rollback would look like without performing the rollback:
 $ oc rollback <deployment_config> --dry-run
 ----
 
+[[executing-commands-inside-a-container-deployments]]
+== Executing Commands Inside a Container
+
+You can add a command to a container, which modifies the container's startup
```

Comment:
Hmm I'll disagree with this one. Getting rid of the comma would ask for a s/which/that, which isn't right here. It's the whole "independent/dependent clause" thing.